### ISSUE TITLE
Will not install on MacOS10.15 

### ISSUE THREAD

**Issue by markosjal** [klep/scanline#40]
This looks like what I need but curious about OS compatibility. It would not install on MacOS10.15

I downloaded from https://github.com/klep/scanline/blob/master/scanline-2.0.pkg?raw=true

Attempting to install on 10.15 I get ...

"This package is incompatible with this version of macOS

The Package is trying to install content on the system volume. Contact The Software manufacturer for assistance."

**Comment by markosjal** [comment]
It looks a lot like this problem!4

https://stackoverflow.com/questions/59399887/macos-catalina-trying-to-install-content-to-the-system-volume

**Comment by klep** [comment]
That's frustrating -- the package is setup to install to /usr/local/bin, so it's odd that Catalina complains about that. You can extract the binary out of it with:

`pkgutil --expand-full scanline-2.0.pkg scanline`

Then just move the `scanline/Payload/Products/usr/local/bin/scanline` file wherever you want it. You might need to also `chmod a+x scanline`

**Comment by markosjal** [comment]
Yea I thought I could gut it that way on Linux now I did it your way on the Mac and did not have to use linux.

It works, made a scan so far so good. I just tested with a Canon imageFormula 215 scanner shared from a Windows host
 using Twain2AirScan, as this particular scanner has no Mac Drivers for current Mac versions, so sharing it this way gets around that Mac Driver issue. I am convinces it would work with any Windows scanner on the other end though.

I am curious are there many scanners still using "drivers" on MacOS or have they all gone to eSCL? I was considering using scanline in a  like solution much like Twain2AirScan, except for a Mac host. It seems it would only be useful for USB scanners though, if most scanners are already eSCL over a network.
